### PR TITLE
ENH expose mamba info

### PR DESCRIPTION
**Pull request by beckermr**
This PR attempts to expose mamba solving info if it is there.

cc @xhochy

### CODE REVIEW

File: conda_forge_tick/status_report.py [hunk 1]
```diff
@@ -199,6 +205,15 @@ def graph_migrator_status(
             for k in sorted(gx2.successors(node))
             if not gx2[k].get("payload", {}).get("archived", False)
         ]
+        if migrator_name in attrs.get("pre_pr_migrator_status", {}) and node in (
+            out["awaiting-pr"] | out["awaiting-parents"]
```
Comment (CJ-Wright) [inline conda_forge_tick/status_report.py:209]: do you need awaiting parents?

Comment (beckermr) [reply conda_forge_tick/status_report.py:209]: likely not - I am also wondering if we should just list solvable as another category. Thoughts?

Comment (CJ-Wright) [reply conda_forge_tick/status_report.py:209]: I would put not solvable in an independent category. Although the messaging may need to state that not solvable doesn't mean there is a problem, just that the CDN doesn't have the pacakge (or there is an actual problem).

Comment (CJ-Wright) [reply conda_forge_tick/status_report.py:209]: We might consider putting the feedstocks that actually error under `bot-error` so we know to look at them more closely (right now I think they just sit as awaiting parents)

Comment (CJ-Wright) [reply conda_forge_tick/status_report.py:209]: We could even set the URL to be the run that they most recently errored on

Comment (beckermr) [reply conda_forge_tick/status_report.py:209]: not in this pr :)

**Comment by beckermr** [comment]
Any other comments here @CJ-Wright?

**Review by CJ-Wright — APPROVED** [review]
LGTM, thank you!

**Comment by xhochy** [comment]
Thank you, this is really helpful!

**Pull request merged**